### PR TITLE
refactor: clean up category share experiment

### PR DESCRIPTION
**Pull request by eddieferrer**
MARS-360

### CODE REVIEW

File: src/components/Thanks/ThanksPageShare.vue [hunk 1]
```diff
@@ -216,10 +201,6 @@ export default {
 			type: Object,
 			default: () => ({}),
 		},
-		simpleSocialShareVersion: {
```
Comment (eddieferrer) [inline src/components/Thanks/ThanksPageShare.vue:219]: Unrelated to this experiment, but this isn't used anywhere

File: src/components/Thanks/ThanksPageShare.vue [hunk 2]
```diff
@@ -2,7 +2,6 @@
 	<div>
 		<ShareStepper
 			:lender-name="lender.firstName"
-			:show-lender-name="showLenderName"
```
Comment (dyersituations) [inline src/components/Thanks/ThanksPageShare.vue:5]: Do we want to remove this prop from `ShareStepper`?

**Review by emuvente — APPROVED** [review]
Looks great! Thanks for cleaning this up!

File: src/components/Thanks/ThanksPageShare.vue (hunk 2 above)
Comment (eddieferrer) [reply src/components/Thanks/ThanksPageShare.vue:5]: No, I want to use the shareStepper for a guest version, where I will use this prop as `false`

**Review by christian14b — APPROVED** [review]
Cool! Thank you for doing this

**Pull request merged**